BitcoinApp/Application/Model/Enums.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Enums.swift
+//  BitcoinApp
+//
+//  Created by Zhassulan Aimukhambetov on 12/5/19.
+//  Copyright © 2019 Zhassulan Aimukhambetov. All rights reserved.
+//
+
+import Foundation
+
+enum Currency: Int {
+    case KZT
+    case USD
+    case EUR
+    
+    var name: String {
+        switch self {
+        case .USD: return "USD"
+        case .EUR: return "EUR"
+        case .KZT: return "KZT"
+        }
+    }
+    
+    var text: String {
```

Comment:
Naming usually is hard so pay additional attention to it. In this case, I suggest using `symbol` or `currencySymbol`.